clients/src/main/java/org/apache/kafka/clients/consumer/internals/CommitRequestManager.java

```diff
@@ -785,6 +805,13 @@ void removeRequest() {
         }
     }
 
+    // Visible for testing
+    Optional<Integer> lastEpochSentOnCommit() {
+        return lastEpochSentOnCommit;
+    }
+
+
+
```

Comment:
Super nit: extra whitespace not needed, right?

Reply:
Sure, removed.

Reply:
Sorry for being the whitespace police 😆